File: app/src/main/java/com/dantsu/thermalprinter/data/UserDetail.java
package com.dantsu.thermalprinter.data;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class UserDetail {
    @JsonProperty("createTime")
    public int createTime;
    @JsonProperty("bioDescription")
    public String bioDescription;
    @JsonProperty("profilePictureUrl")
    public List<String> profilePictureUrl;
}

File: app/src/main/java/com/dantsu/thermalprinter/data/UserDetail.kt
package com.dantsu.thermalprinter.data

import com.fasterxml.jackson.annotation.JsonProperty

class UserDetail {
    @JsonProperty("createTime")
    var createTime = 0

    @JsonProperty("bioDescription")
    var bioDescription: String? = null

    @JsonProperty("profilePictureUrl")
    var profilePictureUrl: List<String>? = null
}